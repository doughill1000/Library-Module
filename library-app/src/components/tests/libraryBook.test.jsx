import React from "react";
import renderer from "react-test-renderer";
import LibraryBook from "../libraryBook";

const libraryBook = {
  book: {
    title: "the book",
    author: "author"
  },
  paperCopies: [],
  audioCopies: []
};

it("Renders library book", () => {
  const component = renderer.create(<LibraryBook libraryBook={libraryBook} />);

  expect(component).toMatchSnapshot();
});
