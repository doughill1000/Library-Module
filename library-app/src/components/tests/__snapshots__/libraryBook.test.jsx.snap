// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders library book 1`] = `
Array [
  <div
    className="row"
>
    <div
        className=" col-8 col-sm-6 col-md-3 col-lg-1"
    >
        <img
            alt="Book Cover"
            src={undefined}
        />
    </div>
    <div
        className="col-12 col-md-8"
    >
        <div>
            Title: 
            the book
             
        </div>
        <div>
            By: 
            author
        </div>
    </div>
    <div
        className="m-2 col-sm-12"
    >
        <a
            href="javascript:void(0)"
            onClick={[Function]}
        >
            Paper
             Copies: 
            0
             
        </a>
        <button
            className="btn btn-success"
            onClick={[Function]}
        >
            Add Copy
        </button>
        <button
            className="btn btn-primary"
            onClick={[Function]}
        >
            Checkout Copy
        </button>
    </div>
    <div
        style={
            Object {
                "display": "inline-block",
              }
        }
    >
        <ul />
    </div>
    <div
        className="m-2 col-sm-12"
    >
        <a
            href="javascript:void(0)"
            onClick={[Function]}
        >
            Audio
             Copies: 
            0
             
        </a>
        <button
            className="btn btn-success"
            onClick={[Function]}
        >
            Add Copy
        </button>
        <button
            className="btn btn-primary"
            onClick={[Function]}
        >
            Checkout Copy
        </button>
    </div>
    <div
        style={
            Object {
                "display": "inline-block",
              }
        }
    >
        <ul />
    </div>
</div>,
  <br />,
]
`;
